phpseclib/Net/SSH2.php

```diff
@@ -3482,7 +3482,7 @@ function _append_log($message_number, $message)
             case self::LOG_REALTIME_FILE:
                 if (!isset($this->realtime_log_file)) {
                     // PHP doesn't seem to like using constants in fopen()
-                    $filename = self::LOG_REALTIME_FILENAME;
+                    $filename = self::LOG_REALTIME_FILE;
```

Comment:
This should never have been made a constant in the first place - rather, it should have been NET_SFTP_LOGGING_REALTIME_FILE. The reason being so that you can define the name yourself.

But overall I don't like the whole logging system anyway and at some point in the master branch I intend to redo the whole thing.